data/json/enchantments.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "type": "enchantment",
+    "id": "ENCH_INVISIBILITY",
+    "condition": "ACTIVE",
+    "ench_effects": [ { "effect": "invisibility", "intensity": 1 } ]
+  },
+  {
+    "type": "enchantment",
+    "id": "ENCH_SHADOW_CLOUD",
+    "condition": "ACTIVE",
+    "ench_effects": [ { "effect": "invisibility", "intensity": 1 } ],
```

Comment:
Normally, a monster with night vision should be able to see you when you are in a shadow cloud. Will this prevent that?

Reply:
Whoops this shouldn't be here at all.